demo-app/src/main/java/cash/z/ecc/android/sdk/demoapp/demos/getaddress/GetAddressFragment.kt

```diff
@@ -6,15 +6,19 @@ import cash.z.ecc.android.bip39.Mnemonics
 import cash.z.ecc.android.bip39.toSeed
 import cash.z.ecc.android.sdk.demoapp.BaseDemoFragment
 import cash.z.ecc.android.sdk.demoapp.databinding.FragmentGetAddressBinding
+import cash.z.ecc.android.sdk.demoapp.ext.requireApplicationContext
```

Comment:
These code changes are external to the mozilla plugin aren't they?

Reply:
This was built on top of another branch to reduce merge conflicts—now that you've merged my other branches to master, I can rebase this branch so the diff will be much smaller.

Reply:
And pushed—now the diff for this branch should be much more clear now.